R/match.R

```diff
@@ -0,0 +1,101 @@
+#' Test for presence of certain groups/levels in a vector
+#'
+#' A validated version of `%in%` intended for use with logical subsetting.
+#' Performs matching between a factor and a set of levels to pose the question
+#' "which of the elements of a factor match these levels?"
+#'
+#' @md
+#' @rdname fct_match
+#' @param f A factor (or character vector).
+#' @param inverse (logical) if `TRUE`, perform a "not match" operation.
```

Comment:
This argument and `fct_exclude()` do not seem compelling to me as they're neither more expression or more succinct than `!fct_match()`

Reply:
Counterpoint: using this within a pipe, e.g.

```r
gss_cat$marital %>% 
  fct_exclude("No answer") %>% 
  table()
```

vs

```r
gss_cat$marital %>% 
  fct_match("No answer") %>% 
  `!` %>% 
  table()
```